portfolio/src/main/webapp/script.js

```diff
@@ -13,16 +13,52 @@
 // limitations under the License.
 
 /**
- * Adds a random greeting to the page.
+ * Gets random valid index of an given array
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+function getRandomIndex(array) {
+  return Math.floor(Math.random() * array.length);
+}
+
+/**
+ * Displays the text in the element in the page with given id
+ */
+function addTexttoPage(text, id){
```

Comment:
meta comment: You asked me about how small functions should be before. I think this function is the borderline. Some would consider this to be slightly too small, but I think it's fine; better to be too small than too big.